drivers/clock_control/nrf5_power_clock.c

```diff
@@ -12,10 +12,16 @@
 #include <clock_control.h>
 #include <misc/__assert.h>
 #include <arch/arm/cortex_m/cmsis.h>
+#include <nrf_power.h>
```

Comment:
Hi,

the path "#include <nrfx/hal/nrf_power.h>" is valid because the CMakeLists.txt code has been written like this:

```
  zephyr_include_directories(nrfx)
  zephyr_include_directories(nrfx/drivers/include)
  zephyr_include_directories(nrfx/hal)
  zephyr_include_directories(nrfx/mdk)
  zephyr_include_directories(.)

```

I will leave it to @anangl to comment on what path is desirable, but for this PR I would assume just <nrf_power.h> makes sense as that is consistent with the other users of the nrfx hal, and support for it was explicitly given when this line of code was added:

`  zephyr_include_directories(nrfx/hal)
`

```
sebo@mach:~/zephyr$ ack "<nrf_"
drivers/usb/device/usb_dc_nrf5.c
22:#include <nrf_clock.h>
23:#include <nrf_power.h>
30:#include <nrf_usbd.h>

drivers/gpio/gpio_nrf5.c
18:#include <nrf_gpiote.h>

drivers/entropy/entropy_nrf5.c
10:#include <nrf_rng.h>

drivers/clock_control/nrf5_power_clock.c
15:#include <nrf_power.h>

arch/arm/soc/nordic_nrf5/nrf52/power.c
12:#include <nrf_power.h>

ext/hal/nordic/nrfx/hal/nrf_ecb.c
35:#include <nrf_ecb.h>

ext/hal/nordic/nrfx/hal/nrf_nvmc.c
37:#include <nrf_nvmc.h>

ext/hal/nordic/nrfx/drivers/nrfx_common.h
40:#include <nrf_peripherals.h>

```

Reply:
I'll wait for anangl's comments for this one.